0.1.15: Can no longer use ExecFork to run an executable from PATH

Since we updated from 1.13 to 1.15 we cannot use ExecFork to execute a command like 'npm' which should be found from the PATH. This seems to be related with a change where the 'executable' property is marked as InputFile - so validation tries to find the file, which obviously does not exist.

```
FAILURE: Build failed with an exception. 

* What went wrong:
Some problems were found with the configuration of task ':ui:runCoverageServerCentral' (type 'ExecFork').
> File '/deployment/ui/npm' specified for property 'commandLine' does not exist. 
> File '/deployment/ui/npm' specified for property 'executable' does not exist.
```

Our task is built up rather dynamically:

```
ext.sharedCoverageServeLogic = { task, List<String> target ->
    configure(task) {
        workingDir "$webappDir"
        group = BasePlugin.BUILD_GROUP
        if (!org.gradle.internal.os.OperatingSystem.current().isWindows()) {
            executable "npm"
        } else {
            executable "npm.cmd"
        }

        args = ["run", "--", "ng", "serve", "--no-watch", "--no-live-reload"] + target
    }
}

task runCoverageServerStandalone(type: com.github.psxpaul.task.ExecFork) {
    sharedCoverageServeLogic(it, ["-c", "cov-standalone"])
    waitForOutput = "Compiled successfully"
    timeout = 240
    stopAfter = triggerStopCoverageStandalone
}
```

At first we thought this has something to do with the upgrade to Gradle 7.0 which happened at the same time, but even after reverting to Gradle 6.8 it does not work. Only after downgrading *both* (0.1.13 and 6.8 respectively) it returns to working.

--- a/src/main/kotlin/com/github/psxpaul/task/AbstractExecFork.kt
+++ b/src/main/kotlin/com/github/psxpaul/task/AbstractExecFork.kt
@@ -44,7 +44,7 @@ abstract class AbstractExecFork : DefaultTask(), ProcessForkOptions {
     @Input
     override abstract fun getEnvironment(): MutableMap<String, Any>
 
-    @InputFile
+    @Input
     override abstract fun getExecutable(): String?
 
     @Input

--- a/src/main/kotlin/com/github/psxpaul/task/ExecFork.kt
+++ b/src/main/kotlin/com/github/psxpaul/task/ExecFork.kt
@@ -1,6 +1,6 @@
 package com.github.psxpaul.task
 
-import org.gradle.api.tasks.InputFile
+import org.gradle.api.tasks.Input
 import org.gradle.process.ProcessForkOptions
 import org.gradle.process.internal.JavaForkOptionsFactory
 import javax.inject.Inject
@@ -20,7 +20,7 @@ open class ExecFork @Inject constructor(forkOptionsFactory: JavaForkOptionsFacto
      * The path to the executable to run
      * @deprecated Use #executable instead
      */
-    @get:InputFile
+    @get:Input
     var commandLine: String?
         get() = executable
         set(value) {

